src/game.py

```diff
@@ -22,7 +22,8 @@ def play(self):
         self.cli.goodbye()
     
     def turn(self):
-        player_choice = self.cli.prompt_user()
+        turn_prompt = f"It's {self.current_player().token}'s turn! Please select a square using 1-9:\n"
```

Comment:
Shouldn't this text string live in the cli file?

Reply:
Yeah probably, let me move it over to a method in the cli class

Reply:
Those changes have been made.